regression/cbmc/r_w_ok_inconsistent_nondet/main.c

```diff
@@ -0,0 +1,21 @@
+#include <assert.h>
+
+char *p;
+
+void test_global_pointer()
+{
+  p = nondet(); 
```

Comment:
:pick: to supress warnings, perhaps forward declare `char* nondet();`?

Reply:
Done